scrape_linkedin/Profile.py

```diff
@@ -150,6 +151,94 @@ def interests(self):
+                                                                                    'lt-line-clamp__raw-line']
+                                                                      })
+                                            rec_text_list = [span.text for span in text_spans]
+                                            rec_text_list = [t.replace('\n', '').replace('  ', '').replace('\r', '') for t in rec_text_list]
+                                            rec_dict['text'] = ''.join(rec_text_list)
+
+                                            recommender = one_or_default(rec, '.pv-recommendation-entity__member')
+                                            if recommender:
+                                                try:
+                                                    rec_dict['recommender']['li_id'] = expr.search(
+                                                        recommender.attrs['href']).group()
+                                                except AttributeError as e:
+                                                    pass
+
+                                                recommender_detail = one_or_default(recommender, '.pv-recommendation-entity__detail')
+                                                if recommender_detail:
+                                                    name = text_or_default(recommender, 'h3')
+                                                    name = name.replace('\n', '')
+                                                    name = name.replace('  ', '')
+                                                    name = name.replace('\r', '')
+                                                    name_splitted = name.split(' ', maxsplit=1)
+                                                    rec_dict['recommender']['firstName'] = name_splitted[0]
+                                                    rec_dict['recommender']['lastName'] = name_splitted[1]
+
+                                                    recommender_ps = recommender_detail.find_all('p', recursive=False)
+                                                    if len(recommender_ps) == 2:
+                                                        recommender_headline, recommender_meta = recommender_detail.find_all(
+                                                            'p')
+                                                        if recommender_headline:
+                                                            headline = text_or_default(recommender, 'h3')
+                                                            headline = headline.replace('\n', '')
+                                                            headline = headline.replace('  ', '')
+                                                            headline = headline.replace('\r', '')
+                                                            rec_dict['recommender']['occupation'] = headline
```

Comment:
As mentioned above, I would suggest removing the headline/occupation.

Reply:
ok